dns.go

```diff
@@ -46,6 +46,41 @@ func (api *API) AddSubdomain(domain string, subdomain string) (*Status, error) {
 	}, nil
 }
 
+// AddZoneRecord - Create zone record
+func (api *API) AddZoneRecord(domain string, subdomain string, record *Record) error {
+	var result string
+	args := []interface{}{
+		api.Username,
+		api.Password,
+		api.CustomerNumber,
+		domain,
+		subdomain,
+		record,
+	}
+	if err := api.XMLRPCClient().Call("addZoneRecord", args, &result); err != nil || result != "OK" {
+		return err
+	}
+
+	results, err := api.GetZoneRecords(domain, subdomain)
+	if err != nil {
+		return err
+	}
+
+	for _, element := range results {
```

Comment:
Are you sure there isn't any other way to accomplish what you need this loop for? The things with the IDs seems a little hacky?

Reply:
I'm not aware of any alternative way. The problem is that their API isn't returning any record_id in the addZoneRecord so the alternative I have found is this hack! Documentation should be improved.

The reason for doing this is to be able to keep track of the record in an internal system